remoteworkitem/trackerquery_repository.go

```diff
@@ -32,208 +30,128 @@ func NewTrackerQueryRepository(db *gorm.DB) *GormTrackerQueryRepository {
 
 // TrackerQueryRepository encapsulate storage & retrieval of tracker queries
 type TrackerQueryRepository interface {
-	CheckExists(ctx context.Context, id string) error
-	Create(ctx context.Context, query string, schedule string, tracker uuid.UUID, spaceID uuid.UUID) (*app.TrackerQuery, error)
-	Save(ctx context.Context, tq app.TrackerQuery) (*app.TrackerQuery, error)
-	Load(ctx context.Context, ID string) (*app.TrackerQuery, error)
-	Delete(ctx context.Context, ID string) error
-	List(ctx context.Context) ([]*app.TrackerQuery, error)
+	repository.Exister
+	Create(ctx context.Context, tq *TrackerQuery) error
```

Comment:
Can you please change the logic so that you pass in a TrackerQuery object and get one out? That is cleaner than modifying what you stuff into the function.

Reply:
done here 
fd04b01